shellscript/bash-linux/README.rst

```diff
@@ -0,0 +1,63 @@
+.. Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
+   International License (the "License"). You may not use this file except in compliance with the
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+
+########################################
+AWS CLI examples for BASH shell on Linux
+########################################
+
+These are examples for the AWS Command Line Interface (CLI) public documentation. All examples have been tested and verified
+to work with AWS CLI version 2. The testing platforms include Amazon Linux 2 and MacOS 10.14, both

```

Comment:
We must include Windows testing, especially as it now supports bash.

Reply:
I'll have to figure out how best to incorporate testing on all three platforms.  This could get time intensive.